### PR TITLE
Add Calendar to Dashboard

### PR DESCRIPTION
**Pull request by danielepo**
Related to issue [Create a Calendar of Events ](https://github.com/HTBox/allReady/issues/377)

Here's a snapshot of the result
image

There are still various filters that could be applied but are still missing:
 - filter by location
 - filter by skills
 - filter by organization

### CODE REVIEW

File: AllReadyApp/Web-App/AllReady/Features/Events/EventByMonthQueryHandler.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+using MediatR;
+
+namespace AllReady.Features.Event
+{
+    public class EventByMonthQueryHandler : IRequestHandler<EventByMonthQuery, IEnumerable<EventViewModel>>
+    {
+        private readonly IAllReadyDataAccess dataAccess;
+
+        public EventByMonthQueryHandler(IAllReadyDataAccess dataAccess)
+        {
```
Comment (mgmccarthy) [inline AllReadyApp/Web-App/AllReady/Features/Events/EventByMonthQueryHandler.cs]: @danielepo, we're trying to phase out the usage of `IAllReadyDataAccess` (I believe it's been marked as obsolete in the latest master branch). For this handler, can you instead inject AllReadyContext and work directly against that? There are examples in the code of how to do this, as well as how to test it. Thanks!

Comment (danielepo) [inline AllReadyApp/Web-App/AllReady/Features/Events/EventByMonthQueryHandler.cs]: absolutelly no problem. I'll fix the code and push the changes soon. 
Just out of curiosity: what's the advantage of phacing out `IAllreadyDataAccess` and working with `AllReadyContext` directly?

any way, it looks like [IAllReadyDataAccess](https://github.com/HTBox/allReady/blob/master/AllReadyApp/Web-App/AllReady/DataAccess/IAllReadyDataAccess.cs) hasn't still been marked as obsolete, am I looking at the wrong place?

Comment (mgmccarthy) [inline AllReadyApp/Web-App/AllReady/Features/Events/EventByMonthQueryHandler.cs]: @danielepo, the PR has yet to be merged (PR #1131)? I know @MisterJames was working on that. To answer your question of why we're phasing out IAllReadyDataAccess, have a read through Issue #622. The title needs to be updated, but in there are the reasons for the team moving away from the IAllReadyDataAccess abstraction.

**Comment by MisterJames** [comment]
:shipit: Looks great @danielepo, thanks for this contribution!

**Pull request merged**